external/fv3fit/tests/reservoir/test_model_adapter.py

```diff
@@ -0,0 +1,89 @@
+import numpy as np
+import xarray as xr
+
+from fv3fit.reservoir.transformers.transformer import Transformer
+from fv3fit.reservoir.domain import RankDivider
+from fv3fit.reservoir.readout import ReservoirComputingReadout
+from fv3fit.reservoir import (
+    Reservoir,
+    ReservoirHyperparameters,
+    HybridReservoirComputingModel,
+)
+from fv3fit.reservoir.model import HybridDatasetAdapter
+
+
+class DoNothingAutoencoder(Transformer):
+    def __init__(self, latent_dims):
+        self._latent_dim_len = latent_dims
+
+    @property
+    def n_latent_dims(self):
+        return self._latent_dim_len
+
+    def encode(self, x):
+        return np.concatenate(x, -1)
+
+    def decode(self, latent_x):
+        return latent_x
```

Comment:
Since the `Transformer`s will be all consistent in returning separate arrays for each output, could just make this test class also do that so that no changes are required after that fix.